### ISSUE TITLE
How to troubleshoot plex client issues

### ISSUE THREAD

**Issue by aptalca** [samcm/synclounge#187]
### Web App

**System Details**

- Operating System: Win10
- Web Browser name and version: Chrome 80

**Plex Media Server details**

- Version: 1.19.1.2645
- Operating System (Windows/Linux/Mac): Linux
- Running in Docker? (Yes/No): Yes
- Location (Local/Remote): Local to the players, but remote to synclounge server
- Remote Access Enabled (Yes/No)?: Yes, through nginx reverse proxy (not direct)

**Plex Client details**

- Name: FireTV, Nvidia shield, Synclounge web player
- Version: latest from stores
- Network connection (Wired/Wifi): Wifi
- Platform: FireTV, Nvidia shield, Synclounge web player

### Self-Hosting

**System Details**

- Operating System: VPS Ubuntu Bionic

**SyncLounge Details**

- Running in Docker? (Yes/No): Yes
- Using a Reverse Proxy (Yes/No): Yes
    If Yes, which Reverse Proxy: Nginx (http only)

### Issue information

**To Reproduce**

Set up the docker with compose (some info redacted):
```
version: "2"
services:
  synclounge:
    image: starbix/synclounge
    container_name: synclounge
    environment:
      - DOMAIN=synclounge.********
      - AUTOJOIN_ENABLED=true
      - AUTOJOIN_SERVERURL=http://synclounge.**********/slserver
      - AUTOJOIN_ROOM=****
      - AUTOJOIN_PASSWORD=*****
      - 'AUTHENTICATION={ "mechanism": "plex", "type": [ "server", "user"], "authorized": [ "***user1", "***user2" ] }'
    restart: unless-stopped
```
Open a mix of regular and incognito browser windows to connect to server multiple times and connect to different devices all through the same plex account.

**Describe your bug/issue**
Please describe the bug/your issue below. If applicable, add screenshots to help explain your problem.

Very first time I set up the docker and connected to 3 clients 1) Synclounge web 2) FireTV 3) Nvidia Shield through incognito windows, everything worked great (it was magical, play, pause, sync, all worked great with all three). Afterwards, I can select the device and enter the room, bur I am no longer able to view or send playback to the Fire tv. Nvidia shield works about 30% of the time for sending playback, but doesn't display the full info in webapp.

By the way, autojoin works great, as well as the authorization. However, I cannot control or view playback on the devices. Sometimes the firetv starts playback from the beginning (instead of resume) 10 min later on its own, as if the command was delayed for a long time and wasn't interpreted correctly when it reached the destination.

The docker logs only show user connections (which seem accurate) and room info. Is there a way to see the logs with regards to connections to players and play/pause states?

In other words, how can I troubleshoot issues connecting to and controlling the players so I can report back here my findings?

Thanks

**Comment by aptalca** [comment]
Follow up based on conversations on discord, in hopes that it helps others:

The connections to plex server by the web app, as well as the initial connection to the plex client can be monitored via the browser console (developer tools) while on synclounge webapp.

Commands sent to the clients can be viewed by turning network logging on in the client settings, and then opening the address displayed on the screen in a browser `http://client-lan-ip:32500/logging`
